### PR TITLE
created the five models: events, users, caterers, hosts, subscribers

### PR DESCRIPTION
**Pull request by velvelshteynberg**
Created the five models: events, users, caterers, hosts, subscribers. 
Also did some config work with Sree. 

Please review and approve. 
Thanks

### CODE REVIEW

**Review by Shane0018 — CHANGES_REQUESTED** [review]
for the columns 
-t.boolean :approved
-t.boolean :require_update
-t.boolean :subscribe
-t.boolean :admin
should they be changed to 
is_approved, does_require_update, is_subscribed, and is_admin?

**Comment by velvelshteynberg** [comment]
Good morning 
I changed 3 of the column names. I didn't change the admin column name because I think it should be called admin because we will just assign a true/false value. 
Please review and approve or let me know what other updates I should work on. 
Thanks

File: db/migrate/20210816014222_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :email
+      t.string :phone
```
Comment (Shoshana01) [inline db/migrate/20210816014222_create_users.rb:6]: shouldnt phone be integer?

**Review by Shoshana01 — CHANGES_REQUESTED** [review]
Looks good to me besides I think the phone number columns should be of type integer

**Comment by velvelshteynberg** [comment]
During development of the data modeling we decided that phone number are strings and that is what I am going based on. 
Please review and approve. 
Thanks

**Review by Shoshana01 — APPROVED** [review]

**Pull request merged**